src/main/kotlin/com/github/avrokotlin/avro4k/annotations.kt

```diff
@@ -8,6 +8,10 @@ import kotlinx.serialization.SerialInfo
 @Target(AnnotationTarget.PROPERTY, AnnotationTarget.CLASS)
 annotation class AvroProp(val key: String, val value: String)
 
+@SerialInfo
+@Target(AnnotationTarget.PROPERTY, AnnotationTarget.CLASS)
+annotation class AvroJsonProp(val key: String, val jsonValue: String)
```

Comment:
We could use `@Language("json")` coming from jetbrains to validate json at least in jetbrains editors, what do you think ?

Reply:
Yes, that would improve the development experience.

Reply:
I've just added it.

Reply:
Can you add it to the `AvroDefault` annotation ? 👼

Reply:
Done